packages/netsuite-adapter/src/suiteapp_file_cabinet.ts

```diff
@@ -169,34 +169,41 @@ const getContent = async (content: unknown): Promise<Buffer> => {
   throw new Error(`Got invalid content value: ${safeJsonStringify(content, undefined, 2)}`)
 }
 
+export const isTooBigFileForSuiteApp = async (
+  change: Change<InstanceElement>,
+  changedInstance: InstanceElement,
+): Promise<boolean> => isAdditionOrModificationChange(change)
+    && isFileInstance(changedInstance)
+    && (await getContent(changedInstance.value.content)).toString('base64').length > MAX_DEPLOYABLE_FILE_SIZE
```

Comment:
`changedInstance` is a redundant arg-
```suggestion
    && (await getContent(getChangeData(change).value.content)).toString('base64').length > MAX_DEPLOYABLE_FILE_SIZE
```